### DISCUSSION TITLE
New version not updating for consequent drafts?

### DISCUSSION THREAD

**Discussion by adam-mrozik** [payloadcms/payload#1245] [Q&A]
Hey,

Another one from my side, not sure if it is a bug or me missing something.

Let's say I create a new Document and save it as draft. When I query it with graphql, it shows correctly:
![image](https://user-images.githubusercontent.com/92296835/195404389-24baeee8-c8b2-4397-8c3c-31441124a723.png)
![image](https://user-images.githubusercontent.com/92296835/195404474-101209aa-d0ed-466a-a4f9-3110223eb21d.png)

So far so good.
Now, let's say I update a content of it and save draft once again. But, querying with Graphql again does not change the state:
![image](https://user-images.githubusercontent.com/92296835/195404654-aa284d21-00ba-4099-a325-6336e3469ef2.png)
![image](https://user-images.githubusercontent.com/92296835/195404734-d9c92845-4e78-44c8-8d43-5c4287df2539.png)

The only way it works is if I manually go to Versions and restore the newest version I have created (which doesn't make sense, because this version was already the newest):
![image](https://user-images.githubusercontent.com/92296835/195405123-552d92aa-6f0c-439d-ae81-1cd423401be7.png)
![image](https://user-images.githubusercontent.com/92296835/195405178-361b9f04-b351-4d72-b006-d60f87f95802.png)

`Publish Changes` works okay, every new publish updates the document queried. For Saved Draft, however, newer drafts do not update the document until you manually select the desired version

**Comment by adam-mrozik** [comment]
Please disregard, I actually forgot it is intended and I can bypass it with `draft:true`

**Reply by jmikrut** [reply]
Ahhh yep! Good catch! I was about to go digging DEEP so thank you for responding here!
